how-to/configure/placement-rules.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Placement Rules
+summary: Learn how to configure Placement Rules.
+category: how-to
+---
+
+# Placement Rules
+
+Placement Rules is an experimental feature introduced since PD v4.0. It is a replica rule system that guides PD to generate corresponding schedules for different types of data. By combining different scheduling rules, you can finely control the attributes such as the number of replicas of any continuous data range, the storage location, the host type, whether to participate in Raft election, and whether to act as the Raft leader.
```

Comment:
storage location, host type, 后面这些跟 number of replicas 是并列关系的，都是一种 attribute。这么改行不行？
```suggestion
Placement Rules is an experimental feature introduced since PD v4.0. It is a replica rule system that guides PD to generate corresponding schedules for different types of data. By combining different scheduling rules, you can finely control the attributes of any continuous data range, such as the number of replicas, the storage location, the host type, whether to participate in Raft election, and whether to act as the Raft leader.
```

Reply:
好的，已改。